pdf_build_src/pandoc_script.py

```diff
@@ -59,10 +91,11 @@ def build_pdf(filename="bids-spec.pdf", logfile="bids-spec_pandoc_log.json"):
     # Add input files to command
     # The filenames in `markdown_list` will ensure correct order when sorted
     cmd += [str(root / index_page)]
-    cmd += [str(root / i) for i in ["../../metadata.yml"] + sorted(markdown_list)]
+    cmd += [str(pandoc_metadata)]
+    cmd += [str(root / i) for i in markdown_list]
```

Comment:
Use the generator here:

```suggestion
    cmd += [str(root / md) for md in _flatten_values(sections)]
```